projects/lookup-showcase/src/app/visual/lookup/lookup-visual.component.html

```diff
@@ -62,6 +64,7 @@
           [ariaLabelledBy]="friendsInputBoxLabel.id"
           [data]="people"
           [debounceTime]="1000"
+          [disabled]="disabled"
```

Comment:
It appears that we are missing the `disabled` input on the input box for these. Could we add that and also ensure that any code examples that we have have both? That is a common thing consumers might do so we should ensure we are properly doing it in our examples.